practiceCourseGenerator/practiceCourseGenerator.py

```diff
@@ -136,9 +129,15 @@ def populateUserDictionary(userList, urlPrefix, urlPost):
 		#SIS IDs and do not belong in our version of Canvas. We do not have 
 		#to generate practice courses for them. Continue iterating.
 		if 'sis_user_id' not in data:
-			logger.info(user + ' is missing SIS_ID')
+			logger.info('Error ' + user + ' is missing SIS_ID')
+			errorCount += 1
 			continue
 		fullName = data['sortable_name'].split(',')
+		#Ran into issue where user did not have last name entered in Canvas.
+		if len(fullName) is not 2:
+			logger.info('Error ' + user + ' only has first or last name entered')
```

Comment:
Paranoia! What if data is malformed and name has bunches of commas? Message could be more generic maybe: "User name format is non-standard"